### PR TITLE
Loop controls and UI mod

### PR DESCRIPTION
**Pull request by timsama**
Loop controls have been added, and I'm trying a new color scheme with help from Christina.

### CODE REVIEW

**Comment by jamie-yu0** [comment]
There's a bug. Track is not playing.

**Comment by timsama** [comment]
I haven't changed any of that code, so it must be the rhombus interface changes

File: app/views/tracks/new.html.erb [hunk 1]
```diff
@@ -73,8 +76,6 @@
   }
 
   var rhomb = new Rhombus();
-  var ptnId = rhomb._song.addPattern();
```
Comment (timgrant) [inline app/views/tracks/new.html.erb:76]: Before, there was a single pattern which was created automatically when rhomb was instantiated. Now, patterns need to be explicitly added by the music application.

Also note how add pattern returns an ID. This is so the music application can have a handle on the active pattern.

File: app/views/tracks/new.html.erb [hunk 2]
```diff
@@ -88,21 +89,33 @@
...
+  document.getElementById("loopcontrol").addEventListener("denoto-updateloopend", 
+    function(e){ 
+    console.log("[LoopControl] Setting loop end to " + e.detail.end);
+    rhomb.setLoopStart(e.detail.end);
+  });
+
   // piano roll events
   document.getElementById("apppianoroll").addEventListener("denoto-writenote", 
     function(e){ 
-    console.log("[PianoRoll] Writing note ID " + e.detail.note._id + " at tick " + e.detail.note._start + ", length " + e.detail.note._length);
```
Comment (timgrant) [inline app/views/tracks/new.html.erb:94]: These lines should not have been changed.

**Comment by timgrant** [comment]
The problem is in the latest merge:

https://github.com/soundsplosion/SoundSplosion/pull/47/files

I'll add some comments to the offending changes. We need to be careful to
validate changes before committing them, and especially before merging them
into develop.

Jeongun has added some tools to facilitate automated testing. I'm hoping
that during tomorrow's meeting we can set up some regression tests to make
it easier to detect problems like this.

**Pull request merged**